app/models/event.rb

```diff
@@ -0,0 +1,19 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  has_many :event_attendances, foreign_key: :attended_event_id, dependent: :destroy
```

Comment:
You don't need to specify the `foreign_key` here since you already have that on the other side of the association.